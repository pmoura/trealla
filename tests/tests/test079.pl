main :-
	N1 is -123456789012345678901234567890,
	N1 == -123456789012345678901234567890,
	write(N1), nl,
	N2 is float(1881676372353657772546715999894626455109783106026821047606410765129148590562263),
	write(N2), nl,
	N3 is gcd(987654321098765432109876543210, 42),
	write(N3), nl.

:- initialization(main).
