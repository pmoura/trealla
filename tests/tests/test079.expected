-123456789012345678901234567890
1.8816763723536579e+78
42
42
9000000000900000000090
122443787781019052100
